packages/perseus-editor/src/components/__tests__/angle-input.test.tsx

```diff
@@ -15,91 +15,52 @@ describe("AngleInput", () => {
         });
     });
 
-    test("calls onChange with new angle (radians)", async () => {
+    test("displays angle in degrees", () => {
         // Arrange
-        const onChangeProps = jest.fn();
-        render(<AngleInput angle={0} onChange={onChangeProps} />, {
+        render(<AngleInput angle={Math.PI / 4} onChange={() => {}} />, {
```

Comment:
Question: What is the `Math.PI / 4` here?

Reply:
That is 45 degrees in radians! I think I just wanted to make it test at an angle.